app/templates/ProfileDetails/editModal.js

```diff
@@ -6,6 +6,7 @@ import ContactEditForm from 'components/organisms/ProfileEditForm/contact';
 import SkillEditForm from 'components/organisms/ProfileEditForm/skills';
 import ExperienceEditForm from 'components/organisms/ProfileEditForm/ExperienceEdit';
 import PortfolioEditForm from 'components/organisms/ProfileEditForm/PortfolioEdit';
+import EditBankForm from 'components/organisms/ProfileEditForm/bankdetails';
```

Comment:
This is not the write file name. Please try again with the right name.

Reply:
Okay ,will correct it.